### ISSUE TITLE
Get different result for same model everytime

### ISSUE THREAD

**Issue by ShiKangdi** [keras-team/keras#10479]
https://gist.github.com/flyyufelix/65018873f8cb2bbe95f429c474aa1294#file-resnet-101_keras-py

I load above resnet-101 model, but get different results everytime. 
I already changed the build function for keras 2.

Keras: 2.1.6
tensorflow: 1.1.0
python: 3.5.5

I also saw the np.random.seed(1337) for reproductivity, but didn't get it. In above code, there is no random process.
